src/auditor.js

```diff
@@ -2,13 +2,7 @@ import { axe } from "axe-core/axe.min.js";
 import report from "./reporter";
 
 export default function () {
-  const options = {
-    "rules": {
-      "color-contrast": { enabled: false },
-    }
-  };
-
-  window.axe.a11yCheck(document, options, (results) => {
+  window.axe.a11yCheck(document, (results) => {
```

Comment:
You can update the Hound config for the repo to kill this and the comment below.